src/api/payment-requests/payment-requests.md

```diff
@@ -829,6 +835,41 @@ descending activity created date.
 }
 {% endjson %}
 
+<a name="accept-payment-condition"></a>
+### Accept Payment Condition For a Payment Request **EXPERIMENTAL**
+
+Accept one of the payment condition listed in the `merchantConditions` field of a Payment Request
+that has status `awaiting-merchant`. Returns a [Payment Activity][] in the response.
```

Comment:
If you are changing the above to Payment Condition this could just describe accepting a payment condition and not specifically reference merchant conditions.

Reply:
I was thinking we just add patronConditions and awaiting-patron when they're valid fields as opposed to emitting the specifics. Otherwise, there is nothing saying when a condition can be accepted.